### PR TITLE
Fix for "DEPRECATION: An addon is trying to access project.nodeModulesPath"

### PR DESCRIPTION
**Pull request by tarponjargon**
ran `ember-cli-update` on the addon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (mike-north) [inline README.md]: You have accidentally committed an un-merged file

File: config/ember-try.js [hunk 2]
```diff
@@ -1,74 +1,70 @@
-module.exports = {
```
Comment (mike-north) [inline config/ember-try.js]: Please remove all whitespace changes to this file.

File: tests/dummy/config/targets.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+const isCI = !!process.env.CI;
+const isProduction = process.env.EMBER_ENV === 'production';
+
+if (isCI || isProduction) {
+  browsers.push('ie 11');
+}
+
 module.exports = {
-  browsers: [
-    'ie 9',
```
Comment (mike-north) [inline tests/dummy/config/targets.js]: We cannot remove support for ie9 without a major version bump

**Comment by tarponjargon** [comment]
Hi Mike - I certainly don't want to give the impression that I know what I'm doing!  I was merely following this recipe to see if I could get rid of a deprecation warning:

[](https://discuss.emberjs.com/t/deprecation-warning-for-addons-future-big-problems/14204/4?u=tarponjargon)

Looks like I failed :(  Further up in that thread broerse mentions that that particular warning comes from a dependency on babel 5.

Sorry if I inconvenienced you!  I like this addon alot - thanks.

**Pull request closed without merging**